assets/xml/objects/object_zo.xml

```diff
@@ -1,28 +1,39 @@
 ﻿<Root>
+    <!-- Object for the Zoras -->
     <File Name="object_zo" Segment="6">
+        <!-- Animations -->
         <Animation Name="gZoraHandsOnHipsTappingFootAnim" Offset="0x598" />
         <Animation Name="gZoraArmsOpenAnim" Offset="0xD48" />
         <Animation Name="gZoraThrowRupeeAnim" Offset="0x219C" />
         <Animation Name="gZoraWalkAnim" Offset="0x2898" />
-        <Texture Name="gZoraBubbleTex" OutName="bubble" Format="ia8" Width="16" Height="16" Offset="0x28B0" />
-        <DList Name="object_zo_DL_0029F0" Offset="0x29F0" />
-        <DList Name="object_zo_DL_002A50" Offset="0x2A50" />
-        <Texture Name="gZoraSplashTex" OutName="splash" Format="i8" Width="16" Height="16" Offset="0x2A70" />
-        <DList Name="object_zo_DL_002BA0" Offset="0x2BA0" />
-        <DList Name="object_zo_DL_002C10" Offset="0x2C10" />
+
+        <!-- Effects -->
+        <Texture Name="gZoraBubblesTex" OutName="bubble" Format="ia8" Width="16" Height="16" Offset="0x28B0" />
```

Comment:
```suggestion
        <Texture Name="gZoraBubblesTex" OutName="zora_bubble" Format="ia8" Width="16" Height="16" Offset="0x28B0" />
```
Is there any particular reason the outNames are missing prefixes in this file?

Reply:
No. I just forgot :sweat_smile: